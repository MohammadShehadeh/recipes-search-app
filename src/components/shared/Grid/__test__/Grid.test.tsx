import renderer from 'react-test-renderer';

import { Grid } from '..';

describe('Grid', () => {
  it('should render to match snapshot', () => {
    const tree = renderer
      .create(
        <Grid>
          <Grid.Col lg={4} md={6} sm={12}>
            Col 1
          </Grid.Col>
          <Grid.Col lg={4} md={6} sm={12}>
            Col 2
          </Grid.Col>
          <Grid.Col lg={4} md={6} sm={12}>
            Col 3
          </Grid.Col>
        </Grid>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
