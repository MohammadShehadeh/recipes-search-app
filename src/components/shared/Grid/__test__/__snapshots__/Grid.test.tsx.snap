// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should render to match snapshot 1`] = `
<div
  className="grid"
>
  <div
    className="col col-sm-12 col-md-6 col-lg-4"
  >
    Col 1
  </div>
  <div
    className="col col-sm-12 col-md-6 col-lg-4"
  >
    Col 2
  </div>
  <div
    className="col col-sm-12 col-md-6 col-lg-4"
  >
    Col 3
  </div>
</div>
`;
